### PR TITLE
Fixed connect with valid SSL certificate

### LINKED ISSUE #79: Connect with valid SSL certificate
Hi,

I can't connect to ROS without `ctx.check_hostname = False`. I have trusted certificate authority signed SSL certificate installed.

```python
from librouteros import connect
import ssl

ctx = ssl.create_default_context()
api = connect(
    username='admin',
    password='',
    host='example.com',
    ssl_wrapper=ctx.wrap_socket,
    port=8729,
)
```

I got error:
```
Traceback (most recent call last):
  File "~/mikrotik/libros.py", line 12, in <module>
    port=8729
  File "~/mikrotik/venv/lib/python3.7/site-packages/librouteros/__init__.py", line 46, in connect
    transport = create_transport(host, **arguments)
  File "~/mikrotik/venv/lib/python3.7/site-packages/librouteros/__init__.py", line 60, in create_transport
    sock = kwargs['ssl_wrapper'](sock)
  File "/Python.framework/Versions/3.7/lib/python3.7/ssl.py", line 423, in wrap_socket
    session=session
  File "/Python.framework/Versions/3.7/lib/python3.7/ssl.py", line 827, in _create
    raise ValueError("check_hostname requires server_hostname")
ValueError: check_hostname requires server_hostname
```

I am using macOS 10.15.4 with Python 3.7.7
Thanks.

### PR DESCRIPTION
**Pull request by Turbid**
fixed #79

### CODE REVIEW

**Comment by luqasz** [comment]
Hi.

Can you confirm that this fix works for you ?

File: librouteros/__init__.py [hunk 1]
```diff
@@ -57,5 +57,5 @@ def connect(host: str, username: str, password: str, **kwargs) -> typing.Type[Ap
 
 def create_transport(host: str, **kwargs) -> SocketTransport:
     sock = create_connection((host, kwargs['port']), kwargs['timeout'], (kwargs['saddr'], 0))
-    sock = kwargs['ssl_wrapper'](sock)
+    sock = kwargs['ssl_wrapper'](sock, server_hostname=host)
```
Comment (luqasz) [inline librouteros/__init__.py:60]: I'd prefer to keep `ssl_wrapper` parameter as a function call with only one argument. Best would be to use:
```python3
from functools import partial

api = connect(
    username='admin',
    password='abc',
    host='some.address.com',
    ssl_wrapper=partial(ctx.wrap_socket, server_hostname='some.address.com')
    port=8729
    )
```
This code will call [wrap_socket](https://docs.python.org/3/library/ssl.html#ssl.SSLContext.wrap_socket) with `server_hostname` set.

**Review by luqasz — CHANGES_REQUESTED** [review]
Please change as in comment. I will do the docs.

File: librouteros/__init__.py (hunk 1 above)
Comment (Turbid) [reply librouteros/__init__.py:60]: Yes, its worked for me. Tnx.

**Comment by Turbid** [comment]
Another way suggested

**Pull request closed without merging**